modules/virtual_network_gateway/variables.tf

```diff
@@ -136,59 +129,95 @@ variable "ip_configuration" {
       subnet_id        = "ID_for_subnet_GatewaySubnet"
     }
   ]
-
+  ```
   EOF
-  type        = list(any)
+  default     = []
+  nullable    = false
+  type = list(object({
+    name                          = optional(string, "vnetGatewayConfig")
```

Comment:
can we have two configs with the same name? if no, I would drop the default value and leave the name property as required + a validation that checks if the name property is unique

Reply:
resolved